data/kernel-options/cgroup-memory/sle15/sp4/profile.yaml

```diff
@@ -0,0 +1,5 @@
+profile:
+  parameters:
+    kernelcmdline:
+      cgroup_enable: memory
+      swapaccount: 1
```

Comment:
This won't work  as intended. `kernelcmdline` is made of keys, so it can be constructed from different sources. To remove existing keys they can be set to `Null`, e.g. in this case:
```yaml
profile:
  parameters:
    cgroup.memory: Null
```
The other keys are superfluous since they're already set in sp2.